cms/templates/cms/toolbar/toolbar_javascript.html

```diff
@@ -7,56 +7,121 @@
+    'csrf': '{{ csrf_token }}',
+    'request': {
+        'language': '{{ request.GET.language }}',
+        'model': '{{ request.toolbar.get_object_model }}',
+        'page_id': '{% if request.current_page.publisher_is_draft %}{{ request.current_page.pk }}{% else %}{{ request.current_page.publisher_public_id }}{% endif %}',
+        'pk': '{{ request.toolbar.get_object_pk }}',
+        'url': '{% language request.toolbar.language %}{% cms_admin_url "cms_page_resolve" %}{% endlanguage %}',
+        'tree': '{% language request.toolbar.language %}{% cms_admin_url "cms_page_changelist" %}{% endlanguage %}'
+    },
+    'lang': {
+        'debug': '{% filter escapejs %}{% blocktrans %}Development version using django CMS {{ cms_version }}, Django {{ django_version }}, Python {{ python_version }}{% endblocktrans %}{% endfilter %}',
+        'cancel': '{% filter escapejs %}{% trans "Cancel" %}{% endfilter %}',
+        'error': '<strong>{% filter escapejs %}{% trans "The following error occured:" %}{% endfilter %}</strong> ',
+        'success': '{% filter escapejs %}{% trans "Action successful... reloading." %}{% endfilter %}',
+        'confirm': '{% filter escapejs %}{% trans "Are you sure you want to delete this plugin?" %}{% endfilter %}',
+        'publish': '{% filter escapejs %}{% trans "Are you sure you want to publish this page?" %}{% endfilter %}',
+        'addPluginPlaceholder': '{% filter escapejs %}{% trans "Plugin will be added here" %}{% endfilter %}',
+        'confirmDirty': '{% filter escapejs %}{% trans "You have unsaved changes." %}{% endfilter %}',
+        'confirmDirtyESC': '{% filter escapejs %}{% trans "Are you sure you want to abandon these changes?" %}{% endfilter %}',
+        'errorLoadingEditForm': '{% filter escapejs %}{% trans "The form could not be loaded. Please check that the server is running correctly." %}{% endfilter %}',
+        'shortcuts': '{% filter escapejs %}{% trans "Shortcuts" %}{% endfilter %}',
+        'shortcutAreas': [
+            {
+                'title': 'CMS Wide Shortcuts',
+                'shortcuts': {
+                    'help': {
+                        'shortcut': '?',
+                        'desc': '{% filter escapejs %}{% trans "Bring up this help dialog" %}{% endfilter %}'
+                    },
+                    'esc': {
+                        'shortcut': 'ESC',
+                        'desc': '{% filter escapejs %}{% trans "Close / cancel" %}{% endfilter %}'
+                    },
+                    'toggle-structure-board': {
+                        'shortcut': 'space',
+                        'desc': '{% filter escapejs %}{% trans "Toggle structure mode" %}{% endfilter %}'
+                    },
+                    'toggle-structure-board-using-hovered-plugin': {
+                        'shortcut': 'shift+space',
+                        'desc': '{% filter escapejs %}{% trans "Toggle structure mode and highlight hovered plugin" %}{% endfilter %}'
```

Comment:
Really should be *hovered-over plugin*, but even that is not very satisfactory. But still better than *hovered*.